client/src/core/props-behavior.js

```diff
@@ -777,7 +818,7 @@ class PropsSelector {
     for (const {stagingProp, boundingBox} of this.props) {
       const {x, y, z, pitch, yaw, roll,
         name, action, description} =
-          stagingProp.userData.originalProp;
+          stagingProp.userData.parent;
```

Comment:
Are you sure about this one?

The idea is that the staging props are a separate map of props dictionaries that are copies of the original ones.

How/Why is the `.userData.parent` from the staging prop the actual OG prop dictionary? Where is this set? You got rid of `stagingProp.userData['originalProp'] = JSON.parse(JSON.stringify(prop.userData.prop));`, but yet nothing seems to replace it.